validate/validate.go

```diff
@@ -144,21 +144,21 @@ func (v *Validator) CheckJSONSchema() (errs error) {
 
 	url, err := JSONSchemaURL(strings.TrimSuffix(v.spec.Version, "-dev"))
 	if err != nil {
-		errs = multierror.Append(errs, err)
+		errs = errors.Join(errs, err)
```

Comment:
I need to check if I'm actually doing these right; I recently made a mistake with stdlib "multi-errors" because (IIRC) `errors.Join` is more an equivalent to `errors.Wrap`, and _nests_ a new error, not making it on the same "level" as other errors when using a loop.